src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataAnalysis.TaintedDataOperationVisitor.cs

```diff
@@ -232,13 +232,28 @@ public override TaintedDataAbstractValue VisitObjectCreation(IObjectCreationOper
                     ProcessTaintedDataEnteringInvocationOrCreation(method, taintedArguments, originalOperation);
                 }
 
+                bool isSanitizingMethod = false;
                 PooledHashSet<string>? taintedTargets = null;
                 PooledHashSet<(string, string)>? taintedParameterPairs = null;
+                PooledHashSet<(string, string)>? sanitizedParameterPairs = null;
+                PooledHashSet<string>? taintedParameterNamesCached = null;
                 try
                 {
-                    if (this.IsSanitizingMethod(method))
+                    var taintedParameterNames = visitedArguments
```

Comment:
Nit: prefer explicit types in this file.

```suggestion
                    IEnumerable<string> taintedParameterNames = visitedArguments
```